### DISCUSSION TITLE
Remove door brace and force door

### DISCUSSION THREAD

**Discussion by ghramsey** [MattFiler/LegendPlugin#97] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/51859157/210161243-2643c546-b429-489b-98b5-d143e80e122c.png)
![image](https://user-images.githubusercontent.com/51859157/210161247-87581aec-1c20-44af-af17-ddf45867484d.png)

I want to remove these door braces and force the two doors in HAB_Airport on the 2nd level. 
One on the interrogation room and the other on the room w/ the corpse and crowbar.

In Tech_Comms I set the doorbrace to door_brace_unlock and moved it's position.  
Then set the door force_open.  This works for Tech_Comms.

These two doors in HAB_Airport however are not responding to setting the to force open.
I tried adding a FrontMechanism = 3 ENUM which should create a door handle and set the door powered on reset.
The door handles do  not appear.   That was a lark.  I was experimenting to see if an alt method of opening it would work.

I tried removing the door brace entities and that resulted in the game creating a static prop of the door brace lacking
the interface prompt to open it.

Any ideas or insights into this will be helpful.
Thank you

*edit*
My temporary solution is to remove the doors setting their positions to 255,255,255 which seems to 
work and no collision barriers are there.   I consider this temp though.

**Comment by MattFiler** [comment]
I'm not too sure on the specifics of this example, but if you continue to have issues finding a solution I'll explore a bit deeper.

In regards to the static door brace when removing the entities, this will be because the game will be falling back to the static Mover instances. Before you delete the entities, click "Edit Movers" and reposition them to somewhere far off in the level.

Eventually there will be better support for editing both Mover and Commands data together at the same time, but the Mover format is super confusing and I haven't spent much time reverse engineering it yet :)

**Comment by ghramsey** [comment, marked as answer]
My plan was to see if I could force the doors open in response to the generator powering up the lights and doors.
The lights are all in triggersquence but none of those contain the door power.  So instead I used the triggerbox
where Ripley reacts to the corpse to unlock the all three doors with braces to unlock. The braces though merely being
told unlock or door_brace_unlock does not remove unlock or move them.  And there is a collision barrier so you cannot
walk past.  I moved them out of range instead.  Now the doors' function is otherwise unhampered and it doesn't look odd.

Thank you for the insight about the staic object and movers. 
I have not messed w/ that at all and looking at the mover instances I'm not sure I want to until something is 
more documented lest I screw something up badly.
